src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -132,6 +132,9 @@ def get_vm_size_completion_list(prefix, action, parsed_args, **kwargs):  # pylin
     register_cli_argument(scope, 'no_auto_upgrade', action='store_true', help='by doing this, extension system will not pick the highest minor version for the specified version number, and will not auto update to the latest build/revision number on any scale set updates in future.')
 
 register_cli_argument('vm image list', 'image_location', location_type)
+register_cli_argument('vm image', 'publisher_name', options_list=('--publisher', '-p'))
+register_cli_argument('vm image', 'offer', options_list=('--offer', '-f'))
+register_cli_argument('vm image', 'sku', options_list=('--sku', '-s'))
```

Comment:
This is probably a stupid question, but does this update impact the `vm image list-offers` and `vm image list-skus` commands as well? If so, then yay!